### PR TITLE
COMMERCE-12232 | Manage UOM on adding SKU/UOM to a Price List modal

### PR DESCRIPTION
**Pull request by 22LucaGiannotta22**
https://liferay.atlassian.net/browse/COMMERCE-12232

Resent from #4200

### CODE REVIEW

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSKUs.testcase [hunk 1]
```diff
@@ -237,6 +237,237 @@ definition {
...
+		task ("When the admin adds a UOM to that SKU") {
+			CommerceJSONSKUs._addCommerceUOM(
+				basePrice = 1,
+				productSKU = "MIN93015",
+				uomKey = "uom1",
+				uomName = "uom1");
+		}
+
+		task ("And opens the modal by clicking the plus button") {
+			Refresh();
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSKUs.testcase:379]: Avoid to use the Refresh if possible

Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSKUs.testcase [hunk 2]
```diff
@@ -237,6 +237,237 @@ definition {
...
+		}
+
+		task ("Then ‘Master Base Promotion’ is checked") {
+			AssertElementPresent(
+				checkboxName = "Master Base Promotion",
+				locator1 = "CommerceEntry#ANY_TABLE_CHECKBOX_CHECKED");
+		}
+
+		task ("When the admin deletes the entry Master Base Promotion") {
+			Refresh();
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSKUs.testcase:439]: Avoid to use the Refresh if possible

Thanks!

**Review by pietrobocale — CHANGES_REQUESTED** [review]
Ask for some changes, look at the left comments

Thanks!

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
relevant failures are not related to the PR. This can be force forwarded

**Comment by 22LucaGiannotta22** [comment]
ci:forward

**Comment by smottal** [comment]
Resent directly here https://github.com/brianchandotcom/liferay-portal/pull/140777 just resolving conflict

**Pull request closed without merging**